report/src/main/kotlin/io/provenance/digitalcurrency/report/stream/EventStreamConsumer.kt

```diff
@@ -0,0 +1,73 @@
+package io.provenance.digitalcurrency.report.stream
+
+import io.provenance.digitalcurrency.report.config.ContractProperties
+import io.provenance.digitalcurrency.report.config.EventStreamProperties
+import io.provenance.digitalcurrency.report.config.logger
+import io.provenance.digitalcurrency.report.domain.CoinMovementRecord
+import io.provenance.digitalcurrency.report.domain.EventStreamRecord
+import io.provenance.eventstream.decoder.moshiDecoderAdapter
+import io.provenance.eventstream.net.okHttpNetAdapter
+import io.provenance.eventstream.stream.flows.blockDataFlow
+import kotlinx.coroutines.flow.collect
+import kotlinx.coroutines.runBlocking
+import org.jetbrains.exposed.sql.transactions.transaction
+import org.springframework.scheduling.annotation.Scheduled
+import org.springframework.stereotype.Component
+import java.util.UUID
+
+@Component
+class EventStreamConsumer(
+    private val contractProperties: ContractProperties,
+    private val eventStreamProperties: EventStreamProperties,
+) {
+
+    private val log = logger()
+    private val eventStreamId = UUID.fromString(eventStreamProperties.id)
+
+    @Scheduled(
+        initialDelayString = "\${event_stream.connect.initial_delay.ms}",
+        fixedDelayString = "\${event_stream.connect.delay.ms}"
+    )
+    fun consumeEventStream() {
```

Comment:
@mtps wanted to make sure this is how you would recommend using new event stream?

Reply:
I believe so, yes. The block flow will come with all transactions, including those with a non-zero code.

Reply:
looks like `BlockResultsResponseResultTxsResults` has a code on it but the `txEvents` helper on `BlockResultsResponseResult` doesnt look at that value at all

does the lib need an update to allow caller to get the codeId back or filter out non ok codes?